Description: Disable freezing unit tests due to non-thread-safety test implementation
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
Forwarded: <not-needed>, disabling these tests is Debian-specific (and needs reconsidering)

--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -4,6 +4,14 @@
 
 find_package(GMock)
 
+include_directories("${CMAKE_SOURCE_DIR}/liblomiri-app-launch")
+#add_definitions ( -DSPEW_UTILITY="${CMAKE_CURRENT_BINARY_DIR}/data-spew" )
+#add_definitions ( -DSESSION_TEMP_FILE="${CMAKE_CURRENT_BINARY_DIR}/libual-test-session-start-temp" )
+#add_definitions ( -DSOCKET_DEMANGLER="${CMAKE_BINARY_DIR}/utils/socket-demangler" )
+#add_definitions ( -DSOCKET_DEMANGLER_INSTALL="${pkglibexecdir}/socket-demangler" )
+#add_definitions ( -DSOCKET_TOOL="${CMAKE_CURRENT_BINARY_DIR}/socket-tool" )
+add_definitions ( -DSNAP_BASEDIR="${CMAKE_CURRENT_SOURCE_DIR}/snap-basedir" )
+
 # Helper test
 
 add_executable (helper-test helper-test.cc)
@@ -22,33 +30,25 @@
 
 # libUAL Test
 
-include_directories("${CMAKE_SOURCE_DIR}/liblomiri-app-launch")
-add_definitions ( -DSPEW_UTILITY="${CMAKE_CURRENT_BINARY_DIR}/data-spew" )
-add_definitions ( -DSESSION_TEMP_FILE="${CMAKE_CURRENT_BINARY_DIR}/libual-test-session-start-temp" )
-add_definitions ( -DSOCKET_DEMANGLER="${CMAKE_BINARY_DIR}/utils/socket-demangler" )
-add_definitions ( -DSOCKET_DEMANGLER_INSTALL="${pkglibexecdir}/socket-demangler" )
-add_definitions ( -DSOCKET_TOOL="${CMAKE_CURRENT_BINARY_DIR}/socket-tool" )
-add_definitions ( -DSNAP_BASEDIR="${CMAKE_CURRENT_SOURCE_DIR}/snap-basedir" )
-
-add_executable (libual-test
-	libual-test.cc
-	mir-mock.cpp)
-target_link_libraries (libual-test gtest_main ${GTEST_MAIN_LIBRARIES} ${LIBUPSTART_LIBRARIES} ${DBUSTEST_LIBRARIES} launcher-static)
-
-add_executable (libual-cpp-test
-	libual-cpp-test.cc
-	mir-mock.cpp)
-target_link_libraries (libual-cpp-test gtest_main ${GMOCK_LIBRARIES} ${LIBUPSTART_LIBRARIES} ${DBUSTEST_LIBRARIES} launcher-static)
-
-add_executable (data-spew
-	data-spew.c)
-target_link_libraries (data-spew ${GLIB2_LIBRARIES})
+#add_executable (libual-test
+#	libual-test.cc
+#	mir-mock.cpp)
+#target_link_libraries (libual-test gtest_main ${GTEST_MAIN_LIBRARIES} ${LIBUPSTART_LIBRARIES} ${DBUSTEST_LIBRARIES} launcher-static)
+
+#add_executable (libual-cpp-test
+#	libual-cpp-test.cc
+#	mir-mock.cpp)
+#target_link_libraries (libual-cpp-test gtest_main ${GMOCK_LIBRARIES} ${LIBUPSTART_LIBRARIES} ${DBUSTEST_LIBRARIES} launcher-static)
+
+#add_executable (data-spew
+#	data-spew.c)
+#target_link_libraries (data-spew ${GLIB2_LIBRARIES})
 
-add_executable (socket-tool
-	socket-tool.c)
+#add_executable (socket-tool
+#	socket-tool.c)
 
-add_test (NAME libual-test COMMAND libual-test)
-add_test (NAME libual-cpp-test COMMAND libual-cpp-test)
+#add_test (NAME libual-test COMMAND libual-test)
+#add_test (NAME libual-cpp-test COMMAND libual-cpp-test)
 
 # App Store Legacy
 
@@ -61,19 +61,19 @@
 
 # Jobs Base Test
 
-add_executable (jobs-base-test
-	jobs-base-test.cpp)
-target_link_libraries (jobs-base-test ${GMOCK_LIBRARIES} ${GTEST_MAIN_LIBRARIES} launcher-static ${DBUSTEST_LIBRARIES})
-
-add_test(NAME jobs-base-test COMMAND jobs-base-test)
+#add_executable (jobs-base-test
+#	jobs-base-test.cpp)
+#target_link_libraries (jobs-base-test ${GMOCK_LIBRARIES} ${GTEST_MAIN_LIBRARIES} launcher-static ${DBUSTEST_LIBRARIES})
+#
+#add_test(NAME jobs-base-test COMMAND jobs-base-test)
 
 # Jobs Systemd Test
 
-add_executable (jobs-systemd
-	jobs-systemd.cpp)
-target_link_libraries (jobs-systemd ${GMOCK_LIBRARIES} launcher-static ${DBUSTEST_LIBRARIES})
-
-add_test(NAME jobs-systemd COMMAND jobs-systemd)
+#add_executable (jobs-systemd
+#	jobs-systemd.cpp)
+#target_link_libraries (jobs-systemd ${GMOCK_LIBRARIES} launcher-static ${DBUSTEST_LIBRARIES})
+#
+#add_test(NAME jobs-systemd COMMAND jobs-systemd)
 
 # Info Watcher ZG
 
@@ -129,13 +129,13 @@
 	COMMAND clang-format -i -style=file
 	application-info-desktop.cpp
 	app-store-legacy.cpp
-	libual-cpp-test.cc
-	libual-test.cc
+#	libual-cpp-test.cc
+#	libual-test.cc
 	list-apps.cpp
 	eventually-fixture.h
 	info-watcher-zg.cpp
-	jobs-base-test.cpp
-	jobs-systemd.cpp
+#	jobs-base-test.cpp
+#	jobs-systemd.cpp
 	libertine-service.h
 	registry-mock.h
 	snapd-info-test.cpp
